pkg/vpcmodel/explainabilityInput.go

```diff
@@ -28,6 +27,101 @@ func (e *ExplanationArgs) Dst() string {
 	return e.dst
 }
 
+const (
+	noErr                        = iota
+	exitNowErr                   // exit now with the error (do not wait until we go over all vpcs)
+	internalNotWithinSubnetsAddr // internal address with not within vpc config's subnet addr - wait until we go over all vpcs
+	noValidInputErr              // string does not represent a valid input w.r.t. this config - wait as above
+)
+
+const (
+	noInternalIP        = iota // nor src neither dst were given as internal IP
+	srcInternalIP              // only src was given as internal ip
+	dstInternalIP              // only dst was given as internal ip
+	srcAndDstInternalIP        // both src and dst given as internal ip
+)
+
+const noValidInputMsg = "does not represent a legal IP address, a legal CIDR or a VSI name"
+const strPrint = "%s"
```

Comment:
why need `strPrint` const?

Reply:
because of lint. It fails without it. @zivnevo ?